site/content/blogs/release-v127.en.md

```diff
@@ -21,7 +21,7 @@ with the AWS Cloud Development Kit (CDK) or YAML Patch overrides. [See detailed
 - **Enable multiple listeners and listener rules**: You can define multiple host-based or path listener rules for [application load balancers](../docs/manifest/lb-web-service.en.md#http)
 or multiple listeners on different ports and protocols for [network load balancers](../docs/manifest/lb-web-service.en.md#nlb).  
   [See detailed section](#enable-multiple-listeners-and-routing-rules-for-load-balancers).
-- **Preview CloudFormation template changes**: You can now run `copilot [noun] package` or `copilot [noun] deploy` commmands with the `--diff` flag to show differences
+- **Preview CloudFormation template changes**: You can now run `copilot [noun] package` or `copilot [noun] deploy` commands with the `--diff` flag to show differences
```

Comment:
```suggestion
- **Preview CloudFormation template changes**: You can now run `copilot [noun] package` or `copilot [noun] deploy` commands with the `--diff` flag to show differences. [See detailed section](#preview-aws-cloudformation-template-changes)
```

Reply:
It is there! In a new line, which I didn't touch 👀